cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -969,19 +996,23 @@ data ChainInformation = ChainInformation
   , slotInEpoch :: Word64
     -- ^ Relative slot number of the tip of the current chain within the
     -- epoch.
+  , blocksLost :: Word64
+    -- ^ Blocks lost due to switch at fork
   }
 
 chainInformation
   :: forall blk. HasHeader (Header blk)
   => ChainDB.NewTipInfo blk
   -> AF.AnchoredFragment (Header blk)
+  -> Word64
   -> ChainInformation
-chainInformation newTipInfo frag = ChainInformation
-    { slots       = unSlotNo $ fromWithOrigin 0 (AF.headSlot frag)
-    , blocks      = unBlockNo $ fromWithOrigin (BlockNo 1) (AF.headBlockNo frag)
-    , density     = fragmentChainDensity frag
-    , epoch       = ChainDB.newTipEpoch newTipInfo
-    , slotInEpoch = ChainDB.newTipSlotInEpoch newTipInfo
+chainInformation newTipInfo frag blocksLost = ChainInformation
+    { slots         = unSlotNo $ fromWithOrigin 0 (AF.headSlot frag)
```

Comment:
Probably better to remove all the alignment between the identified and the `=`.

Its suboptimal to have to change the indention everything one of those identifiers changes or one is added/removed.

Reply:
Removed!